frontend/src/layout/navigation/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -50,10 +51,16 @@ export function Breadcrumbs(): JSX.Element | null {
     return breadcrumbs.length > 0 ? (
         <div className="Breadcrumbs">
             <Breadcrumb breadcrumb={breadcrumbs[0]} />
-            {breadcrumbs.slice(1).map((breadcrumb) => (
+            {breadcrumbs.slice(1).map((breadcrumb, index) => (
                 <React.Fragment key={breadcrumb.name || '…'}>
                     <IconChevronRight className="Breadcrumbs__separator" />
-                    <Breadcrumb breadcrumb={breadcrumb} />
+                    <Breadcrumb
+                        breadcrumb={
+                            index === breadcrumbs.length - 2
```

Comment:
What's going on here? :O

Reply:
This marks the last breadcrumb as the "you are here" one (it's a bit confusingly `index === length - 2` instead of `- 1` because `index` is offset by 1 due to `.slice(1)`.
I agree that this is unclear though. If this adjustment is needed, I suppose it should be taking place in `breadcrumbsLogic` rather than the component?

Reply:
> I suppose it should be taking place in breadcrumbsLogic rather than the component?

+1 so we can give it a name!